test/stdlib/Encoding_Converter_test.rb

```diff
@@ -0,0 +1,364 @@
+    )
+    assert_send_type(
+      "(String) -> nil",
+      Encoding::Converter, :asciicompat_encoding, "UTF-8"
+    )
+    assert_send_type(
+      "(Encoding) -> nil",
+      Encoding::Converter, :asciicompat_encoding, Encoding::UTF_8
+    )
+  end
+
+  def test_search_convpath
+    assert_send_type(
+      "(String, String) -> Array[[Encoding, Encoding]]",
+      Encoding::Converter, :search_convpath,
+      "UTF-8", "EUC-JP"
+    )
+    assert_send_type(
+      "(Encoding, Encoding) -> Array[[Encoding, Encoding]]",
+      Encoding::Converter, :search_convpath,
+      Encoding::UTF_8, Encoding::EUC_JP
+    )
+    assert_send_type(
+      "(Encoding, Encoding, newline: :universal) -> Array[[Encoding, Encoding] | String]",
+      Encoding::Converter, :search_convpath,
+      Encoding::UTF_8, Encoding::EUC_JP, newline: :universal
+    )
+    assert_send_type(
+      "(Encoding, Encoding, newline: :crlf) -> Array[[Encoding, Encoding] | String]",
+      Encoding::Converter, :search_convpath,
+      Encoding::UTF_8, Encoding::EUC_JP, newline: :crlf
+    )
+    assert_send_type(
+      "(Encoding, Encoding, newline: :cr) -> Array[[Encoding, Encoding] | String]",
+      Encoding::Converter, :search_convpath,
+      Encoding::UTF_8, Encoding::EUC_JP, newline: :cr
+    )
+    assert_send_type(
+      "(Encoding, Encoding, universal_newline: true) -> Array[[Encoding, Encoding] | String]",
+      Encoding::Converter, :search_convpath,
+      Encoding::UTF_8, Encoding::EUC_JP, universal_newline: true
+    )
+    assert_send_type(
+      "(Encoding, Encoding, crlf_newline: true) -> Array[[Encoding, Encoding] | String]",
+      Encoding::Converter, :search_convpath,
+      Encoding::UTF_8, Encoding::EUC_JP, crlf_newline: true
+    )
+    assert_send_type(
+      "(Encoding, Encoding, cr_newline: true) -> Array[[Encoding, Encoding] | String]",
+      Encoding::Converter, :search_convpath,
+      Encoding::UTF_8, Encoding::EUC_JP, cr_newline: true
+    )
+    assert_send_type(
+      "(Encoding, Encoding, xml: :text) -> Array[[Encoding, Encoding] | String]",
+      Encoding::Converter, :search_convpath,
+      Encoding::UTF_8, Encoding::EUC_JP, xml: :text
+    )
+    assert_send_type(
+      "(Encoding, Encoding, xml: :attr) -> Array[[Encoding, Encoding] | String]",
+      Encoding::Converter, :search_convpath,
+      Encoding::UTF_8, Encoding::EUC_JP, xml: :attr
+    )
+  end
+end
+
+class Encoding::ConverterTest < Test::Unit::TestCase
+  include TypeAssertions
+
+  # library "pathname", "set", "securerandom"     # Declare library signatures to load
+  testing "::Encoding::Converter"
+
+  def test_double_equal
+    assert_send_type(
+      "(self) -> bool",
+      Encoding::Converter.new("UTF-8", "EUC-JP"), :==, Encoding::Converter.new("UTF-8", "EUC-JP")
+    )
+  end
+
+  # def test_initialize
+  #   assert_send_type  "(*untyped) -> void",
+  #                     Encoding::Converter.new, :initialize
+  # end
```

Comment:
I think `ConverterSingletonTest#test_new` covers the `#initializer` test, so no test is needed here.